tracing-forest/src/printer/pretty.rs

```diff
@@ -2,7 +2,7 @@ use crate::printer::Formatter;
 use crate::tree::{Event, Shared, Span, Tree};
 use crate::Tag;
 use ansi_term::Color;
-use std::fmt::{self, Write};
+use std::fmt::{self, Debug, Write};
```

Comment:
Where is `Debug` used?

Reply:
It's used in a `#[derive]` on `Pretty` on L58

Reply:
Wait, no that's weird it shouldn't be imported at all. Removed.